plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlValidator.java

```diff
@@ -42,10 +41,10 @@ protected void validate(IResource resource, byte[] bytes)
       SaxParserResults parserResults = WebXmlSaxParser.readXml(bytes);
       Map<BannedElement, Integer> bannedElementOffsetMap =
           ValidationUtils.getOffsetMap(bytes, parserResults);
-      String markerId = "com.google.cloud.tools.eclipse.appengine.validation.servletMarker";
       for (Map.Entry<BannedElement, Integer> entry : bannedElementOffsetMap.entrySet()) {
-        createMarker(resource, entry.getKey(), entry.getValue(),
-            markerId, IMarker.SEVERITY_ERROR);
+        BannedElement element = entry.getKey();
+        createMarker(resource, element, entry.getValue(),
```

Comment:
You don't need to pass the markerId and severity separately. createMarker can simply call the getter methods on element